### PR TITLE
Added alums data

### PR DESCRIPTION
**Pull request by cringeyburger**
This PR closes issue #67. Added Alums data for the website.

### CODE REVIEW

**Review by chirag-ghosh — APPROVED** [review]

**Comment by harshkhandeparkar** [comment]
image
We need to have a default dp for users that we don't have gh usernames.

**Review by rohan-b-84 — APPROVED** [review]

File: src/data/alums.json [hunk 1]
```diff
@@ -16,5 +16,286 @@
     "github_username": "sahil-shubham",
     "linkedin_username": "",
     "website_url": "https://sahil-shubham.in"
+  },
+  {
+    "name": "Anjay Goel",
+    "github_username": "AnjayGoel",
+    "linkedin_username": "",
```
Comment (harshkhandeparkar) [inline src/data/alums.json:23]: Is there a reason why linkedin usernames were omitted?

Comment (harshkhandeparkar) [reply src/data/alums.json:23]: @cringeyburger can you open an issue for this? We can merge this PR and handle that later.

**Review by harshkhandeparkar — APPROVED** [review]

File: src/data/alums.json (hunk 1 above)
Comment (cringeyburger) [reply src/data/alums.json:23]: What do you mean by stating that LinkedIn usernames were omitted? 
PS: I opened issue #69 as you requested for default github dp

Comment (harshkhandeparkar) [reply src/data/alums.json:23]: I meant to ask if there is a lore reason why the alums' linkedin usernames were not added but websites were?

**Pull request merged**